new-evaluation.html

```diff
@@ -0,0 +1,563 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>New evaluation</title>
+		<link rel="stylesheet" type="text/css" href="new-evaluation.css">
+	</head>
+	<body style="background-color: #f7f7f7; margin: auto">
+		<header class="header">
+			<img id="logo" src="logo-v2.png">
+			<a style="vertical-align: middle; line-height: 70px" href="evaluations.html">Evaluations</a>
+			<a style="border-bottom: 9px solid #419fe2;vertical-align: middle; line-height: 70px; padding-bottom: 12px" href="new-evaluation.html">New Evaluation</a>
```

Comment:
Same for this. I used the inline styling for testing it out and then I overlooked them. But anyways, I was going to review and improve my code tomorrow, so I'll fix this, as well as the other ones. Thanks for the review! :)